libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -2949,6 +2949,8 @@ def add_bootloader_defaults(f):
 #ifndef HAL_GYROFFT_ENABLED
 #define HAL_GYROFFT_ENABLED 0
 #endif
+
+#define HAL_MAX_CAN_PROTOCOL_DRIVERS 0
```

Comment:
check this doesn't change size of our AP_Periph bootloaders, should be NFC

Reply:
Checked.  NFC.